### PR TITLE
arwenmandos - gas schedules embedded in binary

### PR DESCRIPTION
**Pull request by andrei-marinica**
Additional fix: some test files were getting loaded in the executable too, causing it to crash.

### CODE REVIEW

File: arwenmandos/gasSchedules/scripts/includetoml.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package main
```
Comment (camilbancioiu) [inline arwenmandos/gasSchedules/scripts/includetoml.go:1]: Consider moving this file to`cmd/includetoml/main.go`. All executable program should be in the same place. The file produced by `main()` uses a relative path anyway.

**Review by camilbancioiu — COMMENTED** [review]
Good idea.

File: arwenmandos/gasSchedules/scripts/includetoml.go (hunk 1 above)
Comment (andrei-marinica) [reply arwenmandos/gasSchedules/scripts/includetoml.go:1]: It is only a macro, only used in the `go:generate` in its parent.

Comment (andrei-marinica) [reply arwenmandos/gasSchedules/scripts/includetoml.go:1]: `cmd` contains the public binaries

**Review by camilbancioiu — APPROVED** [review]

**Review by matei-p — APPROVED** [review]

**Pull request merged**